docs/tracingpolicy/README.md

```diff
@@ -0,0 +1,1193 @@
+This filter can be especially useful to track container escape or execution of
+code in a new namespace, for example using user namespace to exploit root-only
+kernel features.
+
+```yaml
+matchNamespaceChanges:
+- operator: In
+  values:
+  - "Mnt"
+```
+
+The `unshare` command, or executing in the host namespace using `nsenter` can
+be used to test this feature. See a
+[demonstration example](https://github.com/cilium/tetragon/blob/c511bf632ee58d401aa0f0cf1df50b33760515e3/crds/examples/match_namespace_changes.yaml)
+of this feature.
+
+#### Capability changes filter
+
+Capability changes filter can be specified under the `matchCapabilityChanges`
+field and provide filtering based on calls that are changing Linux capabilities.
+This filter can be especially useful to detect privilege escalation.
+
+```yaml
+matchCapabilityChanges:
+- type: Effective
+  operator: In
+  isNamespaceCapability: false
+  values:
+  - "CAP_SETUID"
+```
+
+See a [demonstration
+example](https://github.com/cilium/tetragon/blob/c511bf632ee58d401aa0f0cf1df50b33760515e3/crds/examples/match_capability_changes.yaml)
+of this feature.
+
+#### Actions filter
+
+Actions filters are a list of actions that execute when an appropriate selector
+matches. They are defined under `matchActions` and currently, the following
+`action` types are supported:
+- [Sigkill action](#sigkill-action)
+- [Override action](#override-action)
+- [FollowFD action](#followfd-action)
+- [UnfollowFD action](#unfollowfd-action)
+- [CopyFD action](#copyfd-action)
+- [GetUrl action](#geturl-action)
+- [DnsLookup action](#dnslookup-action)
+- [Post action](#post-action)
+
+Note that `Sigkill`, `Override`, `FollowFD`, `UnfollowFD`, `CopyFD` and `Post`
+are executed directly in the kernel BPF code while `GetUrl` and `DnsLookup` are
+happening in userspace after the reception of events.
+
+##### Sigkill action
+
+`Sigkill` action terminates synchronously the process that made the call that
+matches the appropriate selectors from the kernel. In the example below, every
+`__x64_sys_write` system call with a PID not equal to 1 or 0 attempting to
+write to `/etc/passwd` will be terminated.
+
+```yaml
+- call: "__x64_sys_write"
+  syscall: true
+  args:
+  - index: 0
+    type: "fd"
+  - index: 1
+    type: "char_buf"
+    sizeArgIndex: 3
+  - index: 2
+    type: "size_t"
+  selectors:
+  - matchPIDs:
+    - operator: NotIn
+      followForks: true
+      isNamespacePID: true
+      values:
+      - 0
+      - 1
+    matchArgs:
+    - index: 0
+      operator: "Prefix"
+      values:
+      - "/etc/passwd"
+    matchActions:
+    - action: Sigkill
+```
+
+##### Override action
+
+`Override` action allows to modify the return value of call. While `Sigkill`
+will terminate the entire process responsible for making the call, `Override`
+will override the return value that was supposed to be returned with the value
+given in the `argError` field.
```

Comment:
And the parent process will remain running?

Reply:
It completely depends on how the process will handle the return value! I would say mostly yes while many programs might terminate after an unsuccessful syscall that was supposed to succeed.

But I see your point here and will add some text.